sdk/examples/wallet/spammer.rs

```diff
@@ -5,11 +5,9 @@
 //!
 //! Rename `.env.example` to `.env` first, then run the command:
 //! ```sh
-//! cargo run --release --features=wallet --example spammer
+//! cargo run --release --all-features --example spammer
```

Comment:
We made this only wallet on purpose so it's faster

Reply:
just tried ... there's no noticeable difference in speed. and what do you mean by only-wallet? it's part of the default features anyway, so there's no need to explicitly list it in the CLI command, unless we add `--no-default-features`, but then the spammer doesn't work.

Reply:
we could just remove `--all-features`, but I put `--all-features` for the users so they don't have to concern themselves with features when they just start learning about the sdk.

Reply:
I mean without storage feature, but yeah I can also remove it locally if I want it